src/annotator/integrations/test/pdf-test.js

```diff
@@ -218,6 +218,51 @@ describe('annotator/integrations/pdf', () => {
       return document.querySelector('hypothesis-banner');
     }
 
+    describe('content info banner', () => {
+      const contentInfo = {
```

Comment:
Perhaps: `@type` this object as a form of documentation so future devs can quickly grok that this is part of the configuration object.

Reply:
OK. Nothing actually checks JSDoc in test files, but you can still use it purely as documentation.